### PR TITLE
add dob_year and edit antimalarial

### PR DESCRIPTION
**Pull request by ekamau**
1) Added dob_year and 
2) edited antimalarial attributes in visit table - moved from experimental agents to antimalarial treatment, 
as in #178

### CODE REVIEW

**Comment by pipliggins** [comment]
Thanks @ekamau! I've made a couple of minor edits (mostly just adding excludeWhen = "none" where required), but this all looks good to me now.

**Review by pipliggins — APPROVED** [review]

**Comment by pipliggins** [comment]
Left open until we decide how to deal with Washington (with age ranges rather than a single value).

File: isaric/parsers/isaric-rapid.toml [hunk 1]
```diff
@@ -593,10 +593,29 @@
...
+    { field = "overall_exper_cmtype___1", values = { 1 = true } },
+    { field = "overall_exper_cmtype___2", values = { 1 = true } },
+    { field = "daily_exper_cmtype___1", values = { 1 = true } },
+    { field = "daily_exper_cmtype___2", values = { 1 = true } },
+    { field = "exper_cmtype___1", values = { 1 = true } },
+    { field = "exper_cmtype___2", values = { 1 = true } },
+  ]
+
+[visit.treatment_antimalarial_type]
+  combinedType = "set"
```
Comment (sadiekelly) [inline isaric/parsers/isaric-rapid.toml:606]: @ekamau I don't think other antimalarial treatments as indicated in antimal_cmtype, daily_antimal_cmtype and overall_antimal_cmtype have all been included - I couldn't see these in the parser in the main branch and the edits here only include chloroquine and hydroxychloroquine. So these additional ones could be mapped also (primaquine, quinine/mefloquine etc.)

File: isaric/parsers/western-australia.toml [hunk 2]
```diff
@@ -478,6 +472,24 @@
     { field = "overall_antimal_cmyn", ref = "Y/N/NK", description = "Antimalarial agent?" },
     { field = "daily_antimal_cmyn", ref = "Y/N/NK", description = "Antimalarial agent?" },
     { field = "antimal_cmyn", ref = "Y/N/NK", description = "Antimalarial agent on the first day of admission?" },
+    { field = "overall_exper_cmtype___1", values = { 1 = true } },
+    { field = "overall_exper_cmtype___2", values = { 1 = true } },
+    { field = "daily_exper_cmtype___1", values = { 1 = true } },
+    { field = "daily_exper_cmtype___2", values = { 1 = true } },
+    { field = "exper_cmtype___1", values = { 1 = true } },
+    { field = "exper_cmtype___2", values = { 1 = true } },
+  ]
```
Comment (sadiekelly) [inline isaric/parsers/western-australia.toml:458]: same comment as for RAPID re. the other antimalarials listed in *antimal_cmtype variables not yet mapped

Comment (ekamau) [reply isaric/parsers/western-australia.toml:463]: Thanks @sadiekelly  - I have added the extra variables

File: isaric/parsers/isaric-rapid.toml (hunk 1 above)
Comment (ekamau) [reply isaric/parsers/isaric-rapid.toml:612]: Thanks @sadiekelly  - I have added the extra variables

**Comment by ekamau** [comment]
Hi @pipliggins - kindly check the changes to experimental agent .. I moved or changed these to 'treatment_other'. I think we are also meant to move some experimental agent information to metadata.
Thanks!

**Pull request merged**